RctGcs/config.py

```diff
@@ -214,9 +189,7 @@ def qgis_prefix_set(self, value: Any) -> None:
 
     @classmethod
     def __get_qgis_path(cls) -> Path:
-        if "_CONDA_ROOT" not in os.environ:
-            raise RuntimeError("Not a conda environment")
-        pkgs_dir = Path(os.environ['_CONDA_ROOT']).joinpath('pkgs')
+        pkgs_dir = Path.home().joinpath('miniconda3', 'pkgs')
```

Comment:
Explicitly disallows anaconda as opposed to miniconda; disregard if this is desired

Reply:
Fixed in 073c7e70b161e8ed94be64dd9ec874f96ad41076